R/msaGaugeLinearity.R

```diff
@@ -86,7 +86,7 @@ msaGaugeLinearity <- function(jaspResults, dataset, options, ...) {
   if (ready) {
 
     if (length(dataset[[measurements]]) < 2){
-      table2$setError(gettextf("T-Test requires more than 1 measurement. $1%i valid measurement(s) detected in $2%s.", length(dataset[[measurements]]), measurements))
+      table2$setError(gettextf("T-Test requires more than 1 measurement. %1$i valid measurement(s) detected in %2Ss.", length(dataset[[measurements]]), measurements))
```

Comment:
humm, `%2Ss` should be `%2$s`, could you pls review my PR here https://github.com/jasp-stats/jaspQualityControl/pull/256

I think my PR can replace this, thanks(*^_^*)